docs/tools/swift.pod

```diff
@@ -1,30 +1,114 @@
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
+
+To execute a Swift program:
+
+=over
+
+B<swift> program.swift -- <arguments>
+
+=back
+
+To work with the Swift Package Manager:
+
+=over
+
+B<swift> build|package|run|test [options]
+
+=back
+
+To invoke the Swift Compiler:
+
+=over
+
+B<swiftc> [options] <inputs>
+
+=back
+
+The list of supported options is available through the "-help" option:
+
+=over
+
+B<swift> -help
 
-B<swiftc> [B<-emit-object>|B<-emit-assembly>|B<-emit-library>]
-  [-help]
-  B<-o> I<output-file>
-  I<input-filenames>
+B<swift> build -help
 
-The full list of supported options is available via "swift -help" or "swiftc -help".
+B<swiftc> -help
+
+=back
 
 =head1 DESCRIPTION
 
-B<Swift> is a new, high performance systems programming language.  It has a clean
-and modern syntax, and offers seamless access to existing C and Objective-C code
-and frameworks, and is memory safe (by default).
+Swift is a general-purpose programming language built using a modern approach to
+safety, performance, and software design patterns.
+
+The goal of the Swift project is to create the best available language for uses
+ranging from systems programming, to mobile and desktop apps, scaling up to
+cloud services. Most importantly, Swift is designed to make writing and
+maintaining I<correct> programs easier for the developer. To achieve this goal,
+we believe that the most obvious way to write Swift code must also be:
+
+=head2 SAFE
+
+The most obvious way to write code should also behave in a safe manner.
+Undefined behavior is the enemy of safety, and developer mistakes should be
+caught before software is in production. Opting for safety sometimes means Swift
+will feel strict, but we believe that clarity saves time in the long run.
+
+=head2 FAST
+
+Swift is intended as a replacement for C-based languages (C, C++, and
+Objective-C). As such, Swift must be comparable to those languages in
+performance for most tasks. Performance must also be predictable and consistent,
+not just fast in short bursts that require clean-up later. There are lots of
+languages with novel features — being fast is rare.
+
+=head2 EXPRESSIVE
+
+Swift benefits from decades of advancement in computer science to offer syntax
+that is a joy to use, with modern features developers expect. But Swift is never
+done. We will monitor language advancements and embrace what works, continually
+evolving to make Swift even better.
+
+=head1 BUGS
+
+Reporting bugs is a great way for anyone to help improve Swift. The open source
+Swift project uses Jira for tracking bugs. Our Jira instance is located at
+L<https://bugs.swift.org>.
+
+If a bug can be reproduced only within an Xcode project or a playground, or if
+the bug is associated with an Apple NDA, please file a report to Apple’s
+L<bug reporter|https://bugreport.apple.com> instead.
+
+=head1 SEE ALSO
+
+More information and resources are available online.
```

Comment:
I wonder if the non-link text in this section could be either made more informative or removed.

For instance, "More information and resources" is a longer way of saying "See also" and "are available online" is immediately self-evident from the links that follow. Likewise, the "developer.apple.com" link is clearly to an Apple Developer resource, and the "github.com" links are clearly to GitHub. Perhaps the links could be accompanied by some concise description of what's there, or alternatively if that seems redundant, just the links can stand alone.